src/video/video_source_local.rs

```diff
@@ -62,45 +58,31 @@ impl VideoSourceLocalType {
     }
 
     fn usb_from_str(description: &str) -> Option<Self> {
+        let pci_regex_with_hub = Regex::new(
+            r"(?P<interface>[0-9|a-f]+:[0-9|a-f]+:[0-9|a-f]+)\.(?P<usb_hub>\d+-\d)\.(?P<usb_port>\d+)",
+        )
+        .unwrap();
         let pci_regex = Regex::new(
-            r"(?P<interface>[0-9|a-f]+:[0-9|a-f]+:[0-9|a-f]+).(?P<usb_hub>\d+)-(?P<usb_port>\d+)",
+            r"(?P<interface>[0-9|a-f]+:[0-9|a-f]+:[0-9|a-f]+)\.(?P<usb_hub>\d+)-(?P<usb_port>\d+)",
```

Comment:
I would recommend to use the same for pci and pci with hub, but limited until the hub part, for everything else you can do .*

Reply:
@patrickelectric done.

Another option - specially if we were more confident that we are testing all the cases - could be something more descriptive, like:
```Rust
r"^usb-(?P<interface>(([0-9a-fA-F]{2}){1,2}:?){4})\.(usb-)?(?P<usb_hub>(\d[-.]?)+)[-.]+(?P<usb_port>\d[-.]?)+$"
```